copregion/opensource/docker-compose.yml

```diff
@@ -408,6 +408,7 @@ services:
          - PAM_ORGANIZATION_URL=${PAM_ORGANIZATION_URL}
          - PGSQLAPI_PASSWORD=${PGSQLAPI_PASSWORD}
          - PGSQLAPI_USER=${PGSQLAPI_USER}
+         - POD_ID=${POD_NUMBER}
```

Comment:
You are adding this to the PAT Application container, I think from comment it is needed by the `cop` container

Reply:
You're right, I got a bit mixed up there. Thanks